crates/service/src/middleware/attestation_signer.rs

```diff
@@ -30,6 +30,11 @@ pub async fn signer_middleware(
     if let Some(Allocation(allocation_id)) = request.extensions().get::<Allocation>() {
         if let Some(signer) = state.attestation_signers.borrow().get(allocation_id) {
             request.extensions_mut().insert(signer.clone());
+        } else {
+            // Just log this case which is silently passed through next middleware
+            tracing::warn!(warn=%allocation_id,
```

Comment:
having the field named `warn` is not very helpful, instead, we should log it as `allocation_id` for better context

```suggestion
            tracing::warn!(allocation_id,
```